src/components/views/dialogs/devtools/RoomNotifications.tsx

```diff
@@ -39,22 +39,56 @@ export default function RoomNotifications({ onBack }: IDevtoolsProps): JSX.Eleme
                 <h2>{_t("Room status")}</h2>
                 <ul>
                     <li>
-                        {_t("Room unread status: ")}
-                        <strong>{humanReadableNotificationColor(color)}</strong>
+                        {_t(
+                            "Room unread status: <strong>%(roomUnreadStatus)s</strong>",
+                            {
+                                roomUnreadStatus: humanReadableNotificationColor(color),
+                            },
+                            {
+                                strong: (sub) => <strong>{sub}</strong>,
+                            },
+                        )}
                         {count > 0 && (
                             <>
-                                {_t(", count:")} <strong>{count}</strong>
+                                {_t(
+                                    ", count: <strong>%(roomUnreadCount)s</strong>",
+                                    {
+                                        roomUnreadCount: count,
+                                    },
+                                    {
+                                        strong: (sub) => <strong>{sub}</strong>,
+                                    },
+                                )}
                             </>
                         )}
                     </li>
                     <li>
-                        {_t("Notification state is")} <strong>{notificationState}</strong>
+                        {_t(
+                            "Notification state is <strong>%(notificationState)s</strong>",
+                            {
+                                notificationState,
+                            },
+                            {
+                                strong: (sub) => <strong>{sub}</strong>,
+                            },
+                        )}
                     </li>
                     <li>
-                        {_t("Room is ")}
-                        <strong>
-                            {cli.isRoomEncrypted(room.roomId!) ? _t("encrypted ✅") : _t("not encrypted 🚨")}
-                        </strong>
+                        {cli.isRoomEncrypted(room.roomId!)
+                            ? _t(
+                                  "Room is <strong>encrypted ✅</strong>",
+                                  {},
+                                  {
+                                      strong: (sub) => <strong>{sub}</strong>,
+                                  },
+                              )
+                            : _t(
+                                  "Room is <strong>not encrypted 🚨</strong>",
+                                  {},
+                                  {
+                                      strong: (sub) => <strong>{sub}</strong>,
+                                  },
+                              )}
```

Comment:
Can't you move the ternary inside the _t, so you can reuse the substitution?

Reply:
```ts
{_t(cli.isRoomEncrypted(room.roomId!)
    ? "Room is <strong>encrypted ✅</strong>"
    : "Room is <strong>not encrypted 🚨</strong>",
    {},
    {
        strong: (sub) => <strong>{sub}</strong>,
    },
)}
```

Reply:
```
{_t(cli.isRoomEncrypted(room.roomId!)
    ? _td("Room is <strong>encrypted ✅</strong>")
    : _td("Room is <strong>not encrypted 🚨</strong>"),
    {},
    {
        strong: (sub) => <strong>{sub}</strong>,
    },
)}
```

*should work* (untested and other terms and conditions you can imagine)